doctordroid/README.md

```diff
@@ -0,0 +1,41 @@
+# Doctor Droid
+
+## Overview
+
+Doctor Droid is an observability analytics platform that analyzes alerts history and fetches metrics from upstream and downstream services. Use the integration to:
```

Comment:
Could you provide more insight on the value of fetching those metrics? The overview should inform new customers on the below not just existing customers:
1. What does your product do in 1-2 sentences and the value
2. What will customers get from this integration & the value 
3. What type of data will your integration send in and/or pull out?

Reply:
I have updated the description basis your feedback. Do let me know if you think there's any more improvement that can be done here. Thanks!